src/backend/rendering/chtmlexportrendering.cpp

```diff
@@ -32,49 +32,37 @@ CHTMLExportRendering::CHTMLExportRendering(
     , m_addText(addText)
 {}
 
-QString CHTMLExportRendering::renderEntry(KeyTreeItem const & i, CSwordKey * k)
-{
-    if (i.hasAlternativeContent()) {
-        QString ret = i.settings().highlight
+QString CHTMLExportRendering::renderEntry(KeyTreeItem const & item,
+                                          CSwordKey * k) {
+    if (item.hasAlternativeContent()) {
+        QString ret = item.settings().highlight
                       ? "<div class=\"currententry\">"
                       : "<div class=\"entry\">";
-        ret.append(i.getAlternativeContent());
+        ret.append(item.getAlternativeContent());
 
-        if (!i.childList()->isEmpty()) {
-            KeyTree const & tree = *i.childList();
+        if (!item.childList()->isEmpty()) {
+            KeyTree const & tree = *item.childList();
 
             BtConstModuleList const modules(collectModules(tree));
-
             if (modules.count() == 1)
                 // insert the direction into the surrounding div:
                 ret.insert(5,
                            QString("dir=\"%1\" ")
                                .arg(modules.first()->textDirectionAsHtml()));
 
-            Q_FOREACH (KeyTreeItem const * const item, tree)
-                ret.append(renderEntry(*item));
+            Q_FOREACH (KeyTreeItem const * const i, tree)
+                ret.append(renderEntry(*i));
         }
 
         ret.append("</div>");
         return ret; // WARNING: Return already here!
     }
 
-
-    BtConstModuleList const & modules(i.modules());
```

Comment:
What was the reason for removing this reference? This could also have been a separate commit.

Reply:
It just my preference, i do not see much sense in creating shortcut for first level inline getter. I think i ll revert this as there really no strong point for it